examples/gnn_sampler/sampler_context.h

```diff
@@ -29,14 +29,21 @@ limitations under the License.
 
 namespace grape {
 template <typename FRAG_T>
-class SamplerContext : public grape::ContextBase<FRAG_T> {
+class SamplerContext
+    : public VertexDataContext<FRAG_T, std::vector<typename FRAG_T::oid_t>> {
```

Comment:
For Sampler, the context should be TensorContext.

Reply:
TODO: https://github.com/yecol/libgrape-lite/issues/7